src/Automation/Automation/help/New-AzAutomationUpdateManagementAzureQuery.md

```diff
@@ -0,0 +1,161 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Automation.dll-Help.xml
+Module Name: Az.Automation
+online version:
+schema: 2.0.0
+---
+
+# New-AzAutomationUpdateManagementAzureQuery
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
```

Comment:
@jemex please fill out all template sections provided in this file

Reply:
@cormacpayne The template is now filled